alacritty-completions.bash

```diff
@@ -15,7 +15,6 @@ _alacritty()
 
     # If `--command` or `-e` is used, stop completing
     for i in "${!COMP_WORDS[@]}"; do
-        echo "${COMP_WORDS[i]}" >> ./testfile
```

Comment:
Thanks a ton for catching this! I must have forgot removing it after testing the completions.

Reply:
No worries!